dev-python/exam/exam-0.10.6.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit distutils-r1
+
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+DESCRIPTION="Helpers for better testing"
+HOMEPAGE="https://github.com/fluxx/exam"
```

Comment:
Minor styling nitpick but please stick to default skel format.

```
DESCRIPTION=""
HOMEPAGE=""
SRC_URI=""
```

Also please describe the package in git message box with a sentence or two. It helps when viewing by git log history.